nautobot_ssot/integrations/dna_center/jobs.py

```diff
@@ -0,0 +1,108 @@
+"""Jobs for DNA Center SSoT integration."""
+
+from django.urls import reverse
+from django.templatetags.static import static
+from nautobot.extras.choices import SecretsGroupAccessTypeChoices, SecretsGroupSecretTypeChoices
+from nautobot.extras.jobs import BooleanVar, ObjectVar
+from nautobot.extras.models import ExternalIntegration
+from nautobot.tenancy.models import Tenant
+from nautobot.core.celery import register_jobs
+from nautobot_ssot.jobs.base import DataSource, DataMapping
+from nautobot_ssot_dna_center.diffsync.adapters import dna_center, nautobot
+from nautobot_ssot_dna_center.utils.dna_center import DnaCenterClient
+
+
+name = "DNA Center SSoT"  # pylint: disable=invalid-name
+
+
+class DnaCenterDataSource(DataSource):  # pylint: disable=too-many-instance-attributes
+    """DNA Center SSoT Data Source."""
+
+    dnac = ObjectVar(
+        model=ExternalIntegration,
```

Comment:
awesome!
what about using the Controller model instead? (and accessing its ExternalIntegration)